src/test-apps/TestSoftwareUpdate.cpp

```diff
@@ -0,0 +1,818 @@
+/*
+ *
+ *    Copyright (c) 2017 Nest Labs, Inc.
```

Comment:
Should probably be "Copyright (c) 2019 Google LLC".

Reply:
done.